treeherder/webapp/api/resultset.py

```diff
@@ -150,6 +151,26 @@ def trigger_missing_jobs(self, request, project, jm, pk=None):
         except Exception as ex:
             return Response("Exception: {0}".format(ex), 404)
 
+    @detail_route(methods=['post'], permission_classes=[IsStaffOrReadOnly])
+    @with_jobs
+    def trigger_all_talos_jobs(self, request, project, jm, pk=None):
+        """
+        Trigger all the talos jobs in a resultset.
+        """
+        if not pk:
+            return Response({"message": "resultset id required"}, status=400)
+
+        times = int(request.QUERY_PARAMS.get('times', None))
+        if not times:
+            raise ParseError(detail="The 'times' parameter is mandatory for this endpoint")
+
```

Comment:
ParseError raised similar to https://github.com/mozilla/treeherder/blob/master/treeherder/webapp/api/revision.py#L19